packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/NodeComponent.java

```diff
@@ -161,7 +161,8 @@ private Element doRender(VariableManager nodeVariableManager, String targetObjec
         Set<ViewModifier> modifiers = this.computeModifiers(optionalDiagramEvent, optionalPreviousNode, nodeId);
         ViewModifier state = this.computeState(modifiers);
         boolean isPinned = this.isPinned(optionalDiagramEvent, nodeId, optionalPreviousNode);
-        CollapsingState collapsingState = this.computeCollapsingState(nodeId, optionalPreviousNode, optionalDiagramEvent);
+        Optional<Boolean> optionalIsCollapsedByDefault = Optional.ofNullable(nodeDescription.getIsCollapsedByDefaultProvider().test(nodeVariableManager));
```

Comment:
Why `Optional.ofNullable(...)` of something which returns a `boolean`? It can't be `null`, it's just a boolean

Reply:
done